### DISCUSSION TITLE
Empty space at the bottom of a qCard

### DISCUSSION THREAD

**Discussion by AmosSzust** [quasarframework/quasar#14782] [General - Components / Directives / etc] [answered]
Hi

I'm almost certain that this empty space was not there up until recently but I didn't find any card related changes in the latest versions (I'm using 2.10.1), here is the image:
![image](https://user-images.githubusercontent.com/100950035/199613024-1a9b084a-c808-4212-aae5-bf3ec247115a.png)

Looking in the browser's elements tab shows that this empty space is taken by the card div itself but without any components to fill in, just those 3 icons that remain up there. If I remove all the content of the qCard except for one empty qCardSection, I see a very small space but once I add things to the card, this extra space appears.

I have the following at the end of the qcard but removing it does nothing.
```
<q-inner-loading :showing="test">
        <q-spinner-gears size="50px" color="primary" />
</q-inner-loading>
```

Any idea what I might be missing?
Thanks

**Comment by suleimanas** [comment, marked as answer]
hi, could you elaborate more please
maybe use codepen etc.  or tell us the full context of <parent ...> <q-card ...> <q-card-section ...> what classes what attrs used and so on

**Reply by AmosSzust** [reply]
Thanks for your reply. Apparently, it happened because I used a surrounding <q-page **class="column"**>, I wanted the different cards to appear as one column (unlike class="row") and it did the trick but caused that extra space at the bottom.

I now noticed that removing this column class completely, achieves the same effect anyway, without the extra space.
Thanks
